pkg/apiserver/statement/statement.go

```diff
@@ -160,3 +161,28 @@ func (s *Service) nodesHandler(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, nodes)
 }
+
+// @Summary Statement plans
+// @Description Get statement plans
+// @Produce json
+// @Param schema query string true "Statement schema"
+// @Param begin_time query string true "Statement begin time"
+// @Param end_time query string true "Statement end time"
+// @Param digest query string true "Statement digest"
+// @Success 200 {array} statement.Plan
+// @Router /statements/plans [get]
+// @Security JwtAuth
+// @Failure 401 {object} utils.APIError "Unauthorized failure"
+func (s *Service) plansHandler(c *gin.Context) {
```

Comment:
I think we can query the plan along with the detail. Maybe no need to open a new request and invoke a new database query.

Reply:
agree, will update it soon.